### PR TITLE
Update website badges etc.

### PR DESCRIPTION
**Pull request by rempsyc**
Update website badges etc.

### CODE REVIEW

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -3,27 +3,40 @@
...
 to explore and pieces to put together to make the process as smooth as possible.
 
-Start by making a minimal **repr**oducible **ex**ample using the
-[reprex](http://reprex.tidyverse.org/) package. If you haven't heard of or used
-reprex before, you're in for a treat! Seriously, reprex will make all of your
-R-question-asking endeavors easier (which is a pretty insane ROI for the five to
-ten minutes it'll take you to learn what it's all about). For additional reprex
-pointers, check out the [Get help!](https://www.tidyverse.org/help/) resource
+1. Try to see if updating to the latest versions of the *easystats* packages help solve 
+your issue. You can do so with: `easystats::install_latest()`.
```
Comment (IndrajeetPatil) [inline .github/SUPPORT.md:7]: Maybe also mention that they shouldn't forget to refresh the R session before trying again?

File: .github/SUPPORT.md [hunk 2]
```diff
@@ -3,27 +3,40 @@
...
+
+2. If the above doesn't solve your problem, do not email the maintainer. Do not 
+send your data and script by email. We do not have the time or resources to help 
+you troubleshoot by email.
```
Comment (IndrajeetPatil) [inline .github/SUPPORT.md:11]: Second sentence in this paragraph is redundant and can be removed.

Comment (rempsyc) [reply .github/SUPPORT.md]: Thanks, I've made the changes.

**Review by IndrajeetPatil — APPROVED** [review]
Thanks!

**Pull request merged**